<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 30pt &quot;TAKE CUTE&quot;;</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputSenha">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>221</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QLineEdit{ border:none;
	background-color: #CACACA;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputUser">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>221</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QLineEdit{ border:none;
	background-color: #CACACA;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_entrar">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: pink;
    font: bold 10px ;
	text-align:center;

    }</string>
   </property>
   <property name="text">
    <string>Entrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="user_lb">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QLabel{
	font: 87 10pt &quot;Source Code Pro Black&quot;;
}</string>
   </property>
   <property name="text">
    <string>Usu√°rio</string>
   </property>
  </widget>
  <widget class="QLabel" name="senha_lb">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QLabel{
	font: 87 10pt &quot;Source Code Pro Black&quot;;
}</string>
   </property>
   <property name="text">
    <string>Senha</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_l">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
